buildspec/linuxIntegrationTests.yml

```diff
@@ -26,6 +26,10 @@ phases:
             - '>/dev/null pip3 install --upgrade aws-sam-cli'
             - '>/dev/null pip3 install --upgrade awscli'
             - '>/dev/null pip3 install pylint'
+            # SAM CLI is ran inside of a pyenv shim so we need to install the versions with pyenv
+            - '>/dev/null pyenv install 3.6.15'
```

Comment:
Why would this have changed? We've been doing `apt-get install python3.6 python3.7 python3.8` (line 23).

Reply:
Aren't we on a different image now? That's my best guess.

Reply:
Yes it appears to correlate with the image change.